packages/core/i18n/nls.json

```diff
@@ -430,6 +430,7 @@
     "vsx-registry": {
       "downloadCount": "Download count: {0}",
       "errorFetching": "Error fetching extensions.",
+      "errorFetchingHint": "This could be caused by network configuration issues.",
```

Comment:
Minor: Please don't add entries to this file manually. It is automatically generated from source code during release.

Reply:
I was unaware of this. I have undone this change.